### DISCUSSION TITLE
Why do we have the FRAME_TRANSFORMATIONS macro in defs.hpp?

### DISCUSSION THREAD

**Discussion by felker** [PrincetonUniversity/athena#522] [Q&A]
Doesn't seem to be used anywhere @c-white, since the `-t` works by selecting a `rsolver/` file to compile. Should we keep it?

**Comment by c-white** [comment]
This must be a remnant of an old way of doing things. We can get rid of it.

This reminds me -- since I was the first one to add extended physics to the code, I sort of hogged lots of single-character configuration flags. We might want to consider expanding `-s`, `-g`, and `-t` to more meaningful strings, perhaps `-sr`, `-gr`, and `-gr_trans`.

**Reply by felker** [reply]
`-sr,-gr` do seem better, but not sure if it is worth breaking backwards compatibility with peoples' driver scripts. 

`-ft`? `-frame_transform`?

**Comment by c-white** [comment]
Both `-ft` and `-frame_transform` would work, but I'm also not sure how much to worry about compatibility.
